core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -101,30 +111,109 @@ final case class OptionT[F[_], A](value: F[Option[A]]) {
   def getOrElseF[B >: A](default: => F[B])(implicit F: Monad[F]): F[B] =
     F.flatMap(value)(_.fold(default)(F.pure))
 
+  /**
+   * Example:
+   * {{{
+   *  scala> import cats.data.OptionT
+   *
+   *  scala> val optionT: OptionT[List, Int] = OptionT(List(Some(2), None, Some(414), None, None))
+   *  scala> optionT.collect(_ == 2)
+   *  res0: OptionT[List, Boolean] = OptionT(value = List(Some(value = true), None, Some(value = false), None, None))
+   * }}}
+   */
   def collect[B](f: PartialFunction[A, B])(implicit F: Functor[F]): OptionT[F, B] =
     OptionT(F.map(value)(_.collect(f)))
 
   def exists(f: A => Boolean)(implicit F: Functor[F]): F[Boolean] =
     F.map(value)(_.exists(f))
 
+  /**
+   * Example:
+   * {{{
+   *  scala> import cats.data.OptionT
+   *
+   *  scala> val optionT: OptionT[List, Int] = OptionT(List(Some(2), None, Some(414), None, None))
+   *  scala> optionT.filter(el => (el % 2 ==0 ))
```

Comment:
No big deal, but the whitespace on these is a bit weird:

```suggestion
   *  scala> optionT.filter(el => (el % 2 == 0))
```